practices/refactoring.md

```diff
@@ -0,0 +1,83 @@
+---
+slug: refactoring
+stages:
+  - development
+short_description: In computer programming and software design, code refactoring is the process of restructuring existing computer code—changing the factoring—without changing its external behavior.
```

Comment:
Please, rewrite this definition with your own words. Using the direct quote (even though is a good one) hurts this article.

_In computer programming and software design, code refactoring is the process of restructuring existing computer code—changing the factoring—without changing its external behavior_ from https://en.wikipedia.org/wiki/Code_refactoring